spec/reek/smells/smell_detector_spec.rb

```diff
@@ -2,6 +2,18 @@
 require_lib 'reek/smells/smell_detector'
 
 RSpec.describe Reek::Smells::SmellDetector do
+  describe 'exception handling' do
+    describe 'catches all exceptions, enriches them and re-raises them' do
+      it 'adds source file line number to the original exception' do
+        skip
+      end
+
+      it 'adds source file name to the original exception' do
+        skip
+      end
+    end
```

Comment:
Just a bunch of first thoughts here. We can also directly print out the offending code,